docs/tasks/administer-cluster/safely-drain-node.md

```diff
@@ -0,0 +1,84 @@
+---
+assignees:
+- davidopp
+
+---
+ 
+{% capture overview %}
+This page shows how to safely drain a machine, respecting the application-level
+disruption SLOs you have specified using PodDisruptionBudget.
+{% endcapture %}
+
+{% capture prerequisites %}
+
+* We assume you are using Kubernetes release >= 1.5.
```

Comment:
Avoid "we" constructs. Also, specify all the links from root; it's slightly less brittle.

"This task assumes that you have met the following prerequisites:

* You are using Kubernetes version 1.5 or later.
* You have created [PodDisruptionBudget(s)](/docs/admin/disruptions.md) to express the application-level disruption SLOs that you want the system to enforce."